bakery/src/tasks/link-rex.js

```diff
@@ -0,0 +1,75 @@
+const dedent = require('dedent')
+
+const { constructImageSource } = require('../task-util/task-util')
+
+const task = (taskArgs) => {
+  const { inputSource, inputPath, contentSource: maybeContentSource } = taskArgs
+  const imageDefault = {
+    name: 'openstax/cops-bakery-scripts',
+    tag: 'trunk'
+  }
+  const imageOverrides = taskArgs != null && taskArgs.image != null ? taskArgs.image : {}
+  const imageSource = constructImageSource({ ...imageDefault, ...imageOverrides })
+  const contentSource = maybeContentSource != null ? maybeContentSource : 'archive'
+  const bookSlugsUrl = 'https://raw.githubusercontent.com/openstax/content-manager-approved-books/master/approved-book-list.json'
+
+  return {
+    task: 'link rex',
+    config: {
+      platform: 'linux',
+      image_resource: {
+        type: 'docker-image',
+        source: imageSource
+      },
+      inputs: [
+        { name: 'book' },
+        { name: `${inputSource}` }
+      ],
+      outputs: [
+        { name: `${inputSource}` },
```

Comment:
Would this mean two tasks have the same named output? I'm not sure if / how that would work. Would it be better to explicitly name this output and update `build-pdf` to consume it (similar impact to the CLI)?

Reply:
Yes, it amends the xthml docs and puts it back where it found it, so there isn't much disturbance to rest of the flow of the pipeline, was what I was going for. I don't think it's better or worse, I thought it would be less confusing this way and not have to additionally create another output. 

But, I will be updating this so it can make more sense -